src/main/java/Deadline.java

```diff
@@ -0,0 +1,14 @@
+public class Deadline extends Task{
+
+    public String by;
```

Comment:
Should "by" be private or protected, since it should be encapsulated and hidden from the user unless through getter methods?